README.md

```diff
@@ -3,7 +3,7 @@
 [![Travis CI status][travis-shield]][travis-link]
 [![Codecov Coverage status][codecov-shield]][codecov-link]
 
-[travis-shield]: https://travis-ci.org/GoogleCloudPlatform/google-cloud-cpp.svg
+[travis-shield]: https://travis-ci.org/GoogleCloudPlatform/google-cloud-cpp.svg?branch=master
```

Comment:
Could you also update `.travis.yml` to test only the `master` branch as well, please?

Reply:
That would stop folks from:

* Using Travis to build on any branches for their forks.  I for one like to do that, and I use it to test changes to the build scripts.
 
* For folks working on branches out of `GoogleCloudPlatform/google-cloud-cpp` then they will not have any way to test (other than locally) until they create a pull request.

I do not think that is a desirable result.

Reply:
And I looked around for a way to configure this on only the main repository, and could not find a way either.